mlir/include/mlir/Dialect/Rock/Passes.td

```diff
@@ -43,6 +43,11 @@ def RockGemmToGridwisePass : Pass<"rock-gemm-to-gridwise", "::mlir::func::FuncOp
   let dependentDialects = ["rock::RockDialect", "func::FuncDialect"];
 }
 
+def RockAttentionToGridwisePass : Pass<"rock-attention-to-gridwise", "::mlir::func::FuncOp"> {
```

Comment:
Would it be more reasonable to stick this in `gemm-to-gridwise` since it operates at the same "level" of lowering? The same way that utility kernels get handled in `conv-to-gemm`?

Reply:
If we want to do that, I'd like to rename the pass to something like : 'lower-to-gridwise' ?

Reply:
For future (6.1 maybe), 

I think it;d be cleaner to have 4 dialects : 
1) RockHighLevel / RockClient -- similiar to CHLO in XLA.
2) RockGridwise
3) RockBlockwise
4) RockThreadwise

and move these transforms to be conversions really.

Reply:
Yeah, lower-to-gridwise is a good pass.

I'm not sure about the dialect split: `vector` has a lot of progressive lowering within itself, too.